src/sentry/integrations/tasks/link_all_repos.py

```diff
@@ -0,0 +1,88 @@
+import logging
+
+from sentry.integrations.services.integration import integration_service
+from sentry.organizations.services.organization import organization_service
+from sentry.plugins.providers.integration_repository import (
+    RepoExistsError,
+    get_integration_repository_provider,
+)
+from sentry.shared_integrations.exceptions import ApiError
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task, retry
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
+
+def get_repo_config(repo, integration_id):
+    return {
+        "external_id": repo["id"],
+        "integration_id": integration_id,
+        "identifier": repo["full_name"],
+    }
+
+
+@instrumented_task(
+    name="sentry.integrations.tasks.link_all_repos",
```

Comment:
Should these tasks be within the `sentry.integrations.github` ? We have some tasks within their integration (some slack tasks) and other tasks are in `sentry.integrations`. Perhaps we can settle on having integration specific tasks within that integration?

Reply:
sounds good to me 👍